flytekit/core/task.py

```diff
@@ -416,3 +416,30 @@ def wrapper(fn) -> ReferenceTask:
         return ReferenceTask(project, domain, name, version, interface.inputs, interface.outputs)
 
     return wrapper
+
+
+class Echo(PythonTask):
+    """
+    A task that simply echoes the inputs back to the user.
+    The task's inputs and outputs interface are the same.
+    FlytePropeller won't create a pod for this task, it will simply pass the inputs to the outputs.
+    https://github.com/flyteorg/flyte/blob/master/flyteplugins/go/tasks/plugins/testing/echo.go
```

Comment:
Can we include docs for `name`, `inputs`, and `kwargs`?

Reply:
updated